src/components/Organisms/AddressDetailsView/AddressDetailsView.tsx

```diff
@@ -0,0 +1,23 @@
+import { type AddressDetailsViewProps } from "@/utils/types/organisms.types";
+import { AddressInfo } from "@/components/Molecules/AddressInfo/AddressInfo";
+import { AddressOverview } from "@/components/Molecules/AddressOverview/AddressOverview";
+import { TransactionsList } from "@/components/Molecules/TransactionsList/TransactionsList";
+import { AccountCard } from "@/components/Molecules/AccountCard/AccountCard";
+
+export const AddressDetailsView: React.FC<AddressDetailsViewProps> = ({
+    address,
+    chain_names,
+}) => {
+    return (
+        <section className="flex w-full flex-col gap-y-8">
+            <AccountCard address={address} />
+
+            <header className="flex gap-x-16">
+                <AddressOverview address={address} chain_names={chain_names} />
+                <AddressInfo address={address} chain_names={chain_names} />
+            </header>
+
+            <TransactionsList address={address} chain_names={chain_names} />
```

Comment:
Lets keep it consistent and separate transactions as a single organism like the others. `AccountCard`, `AddressOverview` and `AddressInfo` can be the "addressdetailview"

Reply:
I didnt understand this

Reply:
Basically remove the `TransactionsList` from the `AddressDetailView` - We want every component to be some what modular. Lets say if a user only wanted the `TransactionsList` rendered on their page but it would be impossible because the detail overview was attached to the component

Reply:
Try to take a look at how I structured the XYK components and how I implemented them in the templates please

Reply:
Fixed in: 699ed6d